src/app/components/link-custom/link-custom.tsx

```diff
@@ -0,0 +1,54 @@
+import classNames from 'classnames';
+import { Link } from 'react-router-dom';
+import { LinkArrowIcon } from '@/icons';
+
+import styles from './link-custom.module.scss';
+
+type LinkCustomProps = React.DetailedHTMLProps<
+  React.AnchorHTMLAttributes<HTMLAnchorElement>,
+  HTMLAnchorElement
+> & {
+  label: string;
+  href: string;
+  size?: 'small' | 'medium' | 'large';
+  outlined?: boolean;
+  rounded?: boolean;
+  arrow?: boolean;
+  arrowSize?: number;
+  arrowColor?: 'black' | 'white';
+  textRegular?: boolean;
+  className?: string;
+};
+
+export const LinkCustom = ({
+  label,
+  href,
+  size = 'large',
+  outlined = false,
+  rounded = false,
+  arrow = true,
+  arrowSize = 24,
+  arrowColor = 'white',
+  textRegular = false,
+  className = '',
+  ...props
+}: LinkCustomProps) => {
+  const btnClass = classNames(styles.button, styles[size], `stroke:${arrowColor}`, {
```

Comment:
I suppose it will be better to name it as `linkClassName` or `className`. Also you can use `bind` conception here:
```suggestion
  const cx = classNames.bind(styles);

  const btnClass = cx(button, styles[size], `stroke:${arrowColor}`, {
    outlined,
    colored: !outlined,
    text: !arrow,
    regular: textRegular,
    rounded,
```

Reply:
Done